rosidl_typesupport_c/rosidl_typesupport_c-extras.cmake.in

```diff
@@ -4,9 +4,16 @@
 # use the same type of library
 set(rosidl_typesupport_c_LIBRARY_TYPE "@rosidl_typesupport_c_LIBRARY_TYPE@")
 
+include("${rosidl_typesupport_c_DIR}/get_used_typesupports.cmake")
+
 find_package(ament_cmake_core QUIET REQUIRED)
```

Comment:
Nitpick: please reorder the added lines that this line (`ament_cmake_core`) isn't "lost' in the middle. Since it is needed for the below `ament_register_extension()` call I suggest to moved it below the added logic / insert the new logic above this `find_package()` call.

Similar below for C++.

Reply:
Done, see https://github.com/ros2/rosidl_typesupport/pull/73/commits/5d01aa94fad084f5b9f2316a3fbd3e42f8e91c3c.